users/repo.go

```diff
@@ -9,18 +9,18 @@ import (
 )
 
 type UserRepo struct {
-	db  *database.Database
-	ctx context.Context
+	DB  *database.Database
```

Comment:
Does our consumer need direct access to either DB or context? Why did we change this?

Reply:
This is still a problem; consumer of `UserRepo` does not need to access the db or ctx.